### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by ab3lson**
## Proposed change
Certain parts of the integration configuration were made easier to follow.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ecobee.markdown [hunk 1]
```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
...
-1. Click on the **Become a developer** link on the [developer site](https://www.ecobee.com/developers/).
-2. Log in with your ecobee credentials.
-3. Accept the SDK agreement.
-4. Fill in the fields.
-5. Click **save**.
+1. Click on the **Become a developer** link on the [developer site](https://www.ecobee.com/developers/)
+2. Log in with your ecobee credentials
+3. Accept the SDK agreement
+4. Fill in the fields
+5. Click **save**
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Please revert this change. Lists that contain a sentence must end with a period. If one item in the list adheres to this, all items get a full stop.

See: <https://docs.microsoft.com/en-us/style-guide/punctuation/periods>

File: source/_integrations/ecobee.markdown [hunk 2]
```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
...
-4. For authorization method select **ecobee PIN**.
-5. You don't need an Application Icon or Detailed Description.
-6. Click **Create**.
+1. Select the **Developer** option from the hamburger menu on the top-right
+2. Select **Create New**
+3. Complete the form on the right. (Neither of the fields are referenced by Home Assistant)
+    * Name: Must be unique across all ecobee users
+    * Summary: Does not need to be unique
+4. Click *Authorization method* and select **ecobee PIN**
+5. Click **Create**
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Revert the removal of periods.

File: source/_integrations/ecobee.markdown [hunk 3]
```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
...
 
-Under the Name and Summary Section, you will now have an API key. Copy this key and use it in your configuration section below. Click the **X** to close the Developer section.
+Your new application will now appear on the left. Upon clicking on the application, API key will appear on the right. Copy this key and use it in the configuration section below. Click **X** to close the Developer section.
 
-## Configuring the Integration
+## Automatic Configuration
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Our standard is just "Configuration". This slipped in before somehow.

```suggestion
## Configuration
```

File: source/_integrations/ecobee.markdown [hunk 4]
```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
...
 
-To configure the ecobee integration in Home Assistant, you can either use the **Configuration** > **Integrations** menu, or add an entry to `configuration.yaml`.
+1. In the **Configuration** > **Integrations** menu, click **+** and then select `ecobee` from the pop-up menu
+2. In the pop-up box, enter the API key you obtained from ecobee.com
+3. In the next pop-up box, you will be presented with a unique four-character PIN code which you will need to authorize in the [ecobee consumer portal](https://www.ecobee.com/consumerportal/index.html). You can do this by logging in, selecting **My Apps** from the hamburger menu, clicking **Add Application** on the left, entering the PIN code from Home Assistant, clicking **Validate** and then **Add Application** in the bottom right
+4. After authorizing the App on ecobee.com, return to Home Assistant and click **Submit**. If the authorization was successful, a configuration entry will be created and your thermostats and sensors will be available in Home Assistant
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]:
```suggestion
1. In the **Configuration** > **Integrations** menu, click **+** and then select "ecobee" from the pop-up menu.
2. In the pop-up box, enter the API key you obtained from ecobee.com.
3. In the next pop-up box, you will be presented with a unique four-character PIN code which you will need to authorize in the [ecobee consumer portal].(https://www.ecobee.com/consumerportal/index.html). You can do this by logging in, selecting **My Apps** from the hamburger menu, clicking **Add Application** on the left, entering the PIN code from Home Assistant, clicking **Validate** and then **Add Application** in the bottom right.
4. After authorizing the App on ecobee.com, return to Home Assistant and click **Submit**. If the authorization was successful, a configuration entry will be created and your thermostats and sensors will be available in Home Assistant.

```

File: source/_integrations/ecobee.markdown [hunk 5]
```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
...
 
-Under the Name and Summary Section, you will now have an API key. Copy this key and use it in your configuration section below. Click the **X** to close the Developer section.
+Your new application will now appear on the left. Upon clicking on the application, API key will appear on the right. Copy this key and use it in the configuration section below. Click **X** to close the Developer section.
 
-## Configuring the Integration
+## Automatic Configuration
 
-To configure the ecobee integration in Home Assistant, you can either use the **Configuration** > **Integrations** menu, or add an entry to `configuration.yaml`.
+1. In the **Configuration** > **Integrations** menu, click **+** and then select `ecobee` from the pop-up menu
+2. In the pop-up box, enter the API key you obtained from ecobee.com
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Maybe make it a link?

**Review by frenck — APPROVED** [review]
Thanks for the PR @ab3lson!

I've left some comments, could you take a look?

Thanks! 👍

**Review by frenck — CHANGES_REQUESTED** [review]
Wrong button, this PR needs changes. Trying again.

File: source/_integrations/ecobee.markdown (hunk 1 above)
Comment (ab3lson) [reply source/_integrations/ecobee.markdown]: Good to know!

File: source/_integrations/ecobee.markdown (hunk 5 above)
Comment (ab3lson) [reply source/_integrations/ecobee.markdown]: It is now a link

**Review by frenck — APPROVED** [review]
Thanks, @ab3lson 👍 

Happy Hacktoberfest 🎉
